client/mapreduce/src/main/java/org/apache/accumulo/core/client/mapreduce/lib/impl/FileOutputConfigurator.java

```diff
@@ -135,14 +135,15 @@ public static AccumuloConfiguration getAccumuloConfiguration(Class<?> implementi
    * @param conf
    *          the Hadoop configuration object to configure
    * @param compressionType
-   *          one of "none", "gz", "lzo", or "snappy"
+   *          one of "none", "gz", "lzo", "snappy", or "zst"
```

Comment:
I went back and forth between those names.  I had `zstd` everywhere until I saw this: https://github.com/apache/hadoop/blob/trunk/hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/io/compress/ZStandardCodec.java#L232

Reply:
If we don't need to truncate it to `zst` for Hadoop compatibility, then I think `zstd` is better since it's used everywhere pretty consistently:

* https://github.com/facebook/zstd
* http://www.zstd.net
* libzstd-1.3.4-1.fc28.x86_64
* zstd-1.3.4-1.fc28.x86_64
* "Zstd, short for Zstandard, is a fast lossless compression algorithm, targeting real-time compression scenarios at zlib-level compression ratio."
* /usr/lib64/libzstd.so.1

Reply:
Oh, also, we don't name our files following the compressor naming conventions anyway. It's always `.rf` regardless of compression type.

Reply:
Sounds good, "zstd" it is.